File: ApplicationCommon/DTOs/Account/EmployerAccountDTO.cs
using ApplicationCommon.DTOs.Job;
using ApplicationCommon.DTOs.Organization;
using ApplicationCommon.DTOs.User;

namespace ApplicationCommon.DTOs.Account;

public class EmployerAccountDTO
{
    public int Id { get; set; }
    public int UserId { get; set; }
    public UserDTO User { get; set; }
    public int OrganizationId { get; set; }
    public OrganizationDTO Organization { get; set; }
    public List<JobDTO> Jobs { get; set; }
}

File: HelperJobby/client-app/src/DTOs/accountDTOs/EmployerAccountDTO.ts
import {OrganizationDTO} from "../organizationDTOs/OrganizationDTO";
import {JobDTO} from "../jobRelatetedDTOs/JobDTO";
import {UserDTO} from "../userRelatedDTOs/UserDTO";

export interface EmployerAccountDTO {
    id: number;
    userId: number;
    email: string;
    contactNumber: string;
    fullName: string;
    user: UserDTO;
    organizationId: number;
    organization: OrganizationDTO;
    jobs: JobDTO[];
}